Gopkg.toml

```diff
@@ -86,7 +86,8 @@ ignored = ["github.com/Sirupsen/logrus"]
 
 [[constraint]]
   name = "github.com/gravitational/roundtrip"
-  revision = "6a87d116a90b9c4e73bedc19ebc9c17f1b43d826"
+  #revision = "6a87d116a90b9c4e73bedc19ebc9c17f1b43d826"
+  branch = "rjones/context"
```

Comment:
This is temporary until https://github.com/gravitational/roundtrip/pull/17 is merged.